src/gmp_backend.rs

```diff
@@ -60,10 +60,7 @@ from_impl!(|d: i32| from_isize(d as isize), i32);
 from_impl!(|d: i16| from_isize(d as isize), i16);
 from_impl!(|d: i8| from_isize(d as isize), i8);
 iter_impl!();
-serdes_impl!(
-    |b: &Bn| b.0.to_str_radix(16),
-    |s: &str| Mpz::from_str_radix(s, 16)
-);
+serdes_impl!(|b: &Bn| (&b.0).into(), |s: &[u8]| Mpz::from(s));
```

Comment:
code golf champion for Rust serde code 😉

Reply:
Haha